compose/service.py

```diff
@@ -468,7 +467,7 @@ def config_dict(self):
             'options': self.options,
             'image_id': self.image()['Id'],
             'links': self.get_link_names(),
-            'net': self.net.id,
+            'net': self.network_mode.id,
```

Comment:
I didn't change the key from `'net'` here because it'll invalidate people's config hashes, but maybe that's not an issue?